cmd/kpod/common.go

```diff
@@ -31,3 +32,30 @@ func getStore(c *cli.Context) (storage.Store, error) {
 	is.Transport.SetStore(store)
 	return store, nil
 }
+
+func getConfig(c *cli.Context) (*libkpod.Config, error) {
+	config := libkpod.DefaultConfig()
+	if c.GlobalIsSet("config") {
+		err := config.FromFile(c.String("config"))
+		if err != nil {
+			return config, err
+		}
+	}
+	if c.GlobalIsSet("root") {
```

Comment:
My preference would be to keep the config-related flags we're parsing here in this file, and then append them to the global flags list in kpod's main. That way, the flags we're parsing here are also present in this file, and all changes can happen in the same place.